src/components/404page/404page.css

```diff
@@ -0,0 +1,40 @@
+.ErrorPageContainer{
```

Comment:
This is good... however, `flexbox` with `justify-content` and `align-items` would've been better... especially considering the conventions we have been discussing.